packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3027,10 +3073,42 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         type: ContainerMessageType,
         contents: any,
         batch: boolean,
+        contentLength: number,
+        serializedContent?: string,
         appData?: any,
     ) {
         this.verifyNotClosed();
         assert(this.connected, 0x259 /* "Container disconnected when trying to submit system message" */);
+
+        if (this.runtimeOptions.compressionOptions?.minimumSize !== undefined &&
+            serializedContent &&
```

Comment:
Is it Ok that serializedContent  can be empty string and fail this test? Do we care at all, given that we have contentLength - it feels like checking it should be enough (we should ensure somewhere else that minimumSize is positive, and thus contentLength  = 0 is automatically excluded here)

Reply:
I don't think we care that serializedContent can be empty string because of the contentLength check